.prow.yaml

```diff
@@ -1165,7 +1165,7 @@ presubmits:
       - image: quay.io/kubermatic/e2e-kind:with-conformance-tests-v1.0.21
         imagePullPolicy: Always
         command:
-        - "./hack/ci/ci_run_api_e2e.sh"
+        - "./hack/ci/ci-run-api-e2e.sh"
```

Comment:
Thankfully this is only a presubmit, so we don't need to make the filename change backwards compatible (as we would have to do with a postsubmit job).